lib/addons/keda/index.ts

```diff
@@ -108,14 +109,10 @@ export class KedaAddOn extends HelmAddOn {
 function populateValues(helmOptions: KedaAddOnProps): Values {
   const values = helmOptions.values ?? {};
 
-  // Check the workaround for SQS Scalar https://github.com/kedacore/keda/issues/837
   setPath(values, "operator.name",  helmOptions.kedaOperatorName);
-  setPath(values, "podSecurityContext.fsGroup",  helmOptions.podSecurityContextFsGroup);

```

Comment:
Why are these SecContext fields removed? Are these no longer supported?

Reply:
As @shapirov103's comment, it's deprecated, let me add deprecation to these fields and a note it

Reply:
Yes that helps.